### PR TITLE
feat: US138571: LTI - Multi grade dialog

### PR DESCRIPTION
**Pull request by ianlaidlaw**
Adds support for LTI multigrades and interactions with the API.

https://rally1.rallydev.com/#/289692574792d/custom/355050439968?detail=%2Fuserstory%2F632812019667&fdp=true

Related prs:
activities: https://github.com/BrightspaceHypermediaComponents/activities/pull/2830
lms: https://github.com/Brightspace/lms/pull/23528

### CODE REVIEW

File: src/activities/associateGrade/AssociateGradeEntity.js [hunk 1]
```diff
@@ -196,6 +207,10 @@ export class AssociateGradeEntity extends Entity {
 		return this._setNewGradeProperty(GRADE_NAME, gradeName);
 	}
 
+	setNewGradeId(newGradeId) {
```
Comment (ianlaidlaw) [inline src/activities/associateGrade/AssociateGradeEntity.js:210]: no longer needed

File: src/activities/associateGrade/AssociateGradeEntity.js [hunk 2]
```diff
@@ -69,7 +70,17 @@ export class AssociateGradeEntity extends Entity {
...
+	gradeItemId() {
+		return this._entity && this._entity.properties && this._entity.properties.gradeItemId;
+	}
+
+	originalGradeItemId() {
+		return this._entity && this._entity.properties && this._entity.properties.originalGradeItemId;
+	}
+
 	canEditGradebookStatus() {
+		if (!this._entity) return false;
```
Comment (ianlaidlaw) [inline src/activities/associateGrade/AssociateGradeEntity.js:82]: shouldn't be needed

Comment (kristian-torres) [reply src/activities/associateGrade/AssociateGradeEntity.js:82]: yeah, your return statement will return false.

File: src/activities/associateGrade/AssociateGradeEntity.js (hunk 1 above)
Comment (kristian-torres) [reply src/activities/associateGrade/AssociateGradeEntity.js:210]: 👍

**Review by kristian-torres — APPROVED** [review]

**Pull request merged**